README.md

```diff
@@ -16,23 +16,21 @@ Currently unsorted, the following is intended to function as a discussion guide
 to developers/contributors.
 
 ### In Progress
-- [ ] Toxcore
-    - [ ] 100% unit testing
-    - [ ] Make ToxAV stateless
-    - [ ] Allow a single toxcore instance to handle multiple keypairs (or 'clients')
-    - [ ] Consistent naming scheme throughout toxcore
-    - [X] Make toxcore stateless
-- [ ] Messenger
-    - [ ] Improve group chat implementation
-    - [ ] Improve A/V implementation
-    - [ ] Multiple device support
+- [ ] 100% unit testing
+- [ ] Allow a single toxcore instance to handle multiple keypairs (or 'clients')
+- [ ] Consistent naming scheme throughout toxcore
+- [ ] Improve A/V implementation
+- [ ] Improve group chat implementation
+- [ ] Make ToxAV stateless
+- [ ] Multiple device support
 
 ### Done
-- [X] Create Toxcore
 - [X] Create DHT
+- [X] Create Messenger
 - [X] Create Onion
+- [X] Create Toxcore
```

Comment:
If you're adding these here, the order should be chronological. Or is there a better order that this into that I don't know?